docs/source/conf.py

```diff
@@ -51,53 +47,75 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+# html_static_path   = ['_static']
+html_theme         = 'bootstrap'
+html_favicon       = 'favicon.gif'
+html_logo          = 'favicon.gif'
+html_theme         = 'bootstrap'
+html_theme_options = {
+    'navbar_site_name': "Open MMBN",
+    'navbar_links': [
+        ("Contributing", "https://github.com/TheMaverickProgrammer/battlenetwork/wiki/Contributing", True)
+    ],
+    'source_link_position': "exclude",
+    'navbar_class': "navbar",
+    'navbar_fixed_top': "true",
+    'navbar_sidebarrel': False,  # Exhale does not support next / previous well
+    'bootswatch_theme': "sandstone",
+    'bootstrap_version': "3",
+}
 
 # -- Breathe setup -----------------------------------------------------------
-breathe_projects = { "C++ MMBN Project": "xml" }
-breathe_default_project = "C++ MMBN Project"
+# NOTE: see also, OUTPUT_DIRECTORY and XML_OUTPUT_DIRECTORY in Doxyfile.
+# If those change, this must update.
+this_file_dir = os.path.dirname(os.path.abspath(__file__))
+breathe_projects = {project: os.path.join(this_file_dir, '_doxygen', 'xml')}
+breathe_default_project = project
```

Comment:
updates in Doxyfile: `{OUTPUT_DIRECTORY=_doxygen}/{XML_OUTPUT_DIRECTORY=xml}`, re-use `project` variable defined above.